### ISSUE TITLE
ScopeKeyData's dependency graph is confusing

### ISSUE THREAD

**Issue by phillipskevin** [canjs/can-view-scope#156]
The observation created internally by `scope-key-data`:
https://github.com/canjs/can-view-scope/blob/9064688f627967313689ad9904743f0841613c56/scope-key-data.js#L90

is in the wrong place in the dependency graph. This code:

```js
import DefineMap from "can-define/map/map";
import Scope from "can-view-scope";
import computeData from "can-view-scope/compute_data";
import canDebug from "can-debug";

var vm = new DefineMap({
    message: "hello"
});

var scope = new Scope(vm);
var scopeKeyData = computeData(scope, "message");

canDebug.drawGraph(scopeKeyData);
```

Creates this graph:

![image](https://user-images.githubusercontent.com/5851984/37849608-577e5bf6-2ea6-11e8-8d6a-2ad33aaafd3d.png)

This Observation should really be in between the DefineMap and the ScopeKeyData.

**Comment by phillipskevin** [comment]
This only happens for "fastPath". Otherwise, the graph makes more sense:

![image](https://user-images.githubusercontent.com/5851984/37850212-5a1370a2-2ea8-11e8-98c3-3d6e5a937596.png)

**Comment by phillipskevin** [comment]
Going to change this to show the "slow-path" graph all the time:

![image](https://user-images.githubusercontent.com/5851984/37855564-dcb1c99a-2ebd-11e8-8b62-e5224447e2e1.png)

It won't be exactly correct for "fast-path", but this won't make it less useful to users.

**Comment by phillipskevin** [comment]
I have this working on https://github.com/canjs/can-view-scope/commit/a6cb995a8552cfd1be84c0f80fc7ad2b7ef66082. I am working on handling nested properties and making sure the graph changes as dependencies change.

**Comment by phillipskevin** [comment]
closed by #157
